File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/Patterns/Pattern2.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here
        int n=scn.nextInt();
        for(int i=1;i<=n;i++)
        {
            for(int j=n-i+1;j>=1;j--)
            System.out.print("*\t");
        System.out.println();
            
        }
        
    }
}

File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/Patterns/pattern2.cpp
#include <iostream>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    
   //write your code here
   for(i=0;i<n;i++)
   {
       for (j  = n; j >0; j--)
       {
           if(j<n-1)
           {
           cout << "";
           }
           else 
           {
            cout << "*\t";
            cout << "\n" ;
           }
       }
       cout << "\n" ;
   }
   return 0;
}
